.idea/jarRepositories.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we still need to commit .gradle directory, or can IntelliJ just generate it from gradle?

Reply:
Removed